docs_src/components/Icon.svelte

```diff
@@ -81,7 +81,7 @@
   <Button className="margin-m--r" type="cancel" on:click={decrement}>
     <Icon name="angleleft" width={18} />
   </Button>
-  <p>Size: {size}px</p>
+  <p>Size: <input type="number" bind:value={size} style="width: 45px;" />px</p>
```

Comment:
Nååå du var ikke så vild med at klikke 82 gange for at få en pixel 100 på ikonerne :p